### PR TITLE
  Deleted "garbage" files that got erroneously merged in the past.

### PR DESCRIPTION
**Pull request by mahrossi**
Discovered some files that should not have made into the repository. I think they came from @litman90 (according to blame ;) ). I deleted them. Hope it is a smooth merge.

### CODE REVIEW

File: examples/dev_tests/lammps/h2o-geop-fixatoms/bfgstrm-0fixed/input.xml~HEAD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!-- REGTEST
```
Comment (mahrossi) [inline examples/dev_tests/lammps/h2o-geop-fixatoms/bfgstrm-0fixed/input.xml~HEAD:1]: I will fix this before merging.

**Comment by mahrossi** [comment]
We do _not_ understand why this is still trying to run circleCI instead of our actions. This is very strange!!

**Comment by mahrossi** [comment]
According to myself this is now ready to be merged and should/could be merged without much reviewing (pretty basic restructuring of non-essential files, plus making sure that the CI works properly)

**Comment by litman90** [comment]
Everything looks fine.

**Pull request merged**